drivers/base/component.c

```diff
@@ -506,8 +507,10 @@ static int component_bind(struct component *component, struct master *master,
 		devres_release_group(component->dev, NULL);
 		devres_release_group(master->dev, NULL);
 
-		dev_err(master->dev, "failed to bind %s (ops %ps): %d\n",
-			dev_name(component->dev), component->ops, ret);
+		if (ret != -EPROBE_DEFER) {
```

Comment:
Please remove the extra braces.

Reply:
aren't the braces required due to the `dev_err` going across 2 lines?

Reply:
C doesn't need them - delete them, unless you are saying that the Linux style requires them.

Reply:
ah, thought they were required by C(I don't really do a lot of C coding day to day), updated just now